IHP/IDE/CodeGen/ViewGenerator.hs

```diff
@@ -171,6 +171,7 @@ buildPlan' schema config =
                 <> "                </thead>\n"
                 <> "                <tbody>{forEach " <> pluralVariableName <> " render" <> singularName <> "}</tbody>\n"
                 <> "            </table>\n"
+                <> "            { renderPagination pagination }\n"
```

Comment:
Sounds good, thanks. This is a habit I have developed from using languages like CalcCheck (https://calccheck.mcmaster.ca) where spaces are needed to denote different symbols, since they have full Unicode support for all symbols in the language (proof by emoji is my personal favourite 😉)